src/js/_store/modules/articles.js

```diff
@@ -124,6 +139,23 @@ export default {
       axios(rootGetters['auth/token'])({
         method: 'GET',
         url: '/article',
+      }).then(res => {
+        const payload = {
+          articles: res.data.articles,
+        };
+        const lastPage = res.data.meta.last_page;
+        const firstPage = res.data.meta.current_page;
```

Comment:
[imo]
オブジェクトとして定義すればmutationの`doneGetLastArticles`と`doneGetFirstArticles`は一つにまとめられそうです！

```suggestion
        const articleMeta = {
          firstPagination: res.data.meta.current_page,
          lastPagination: res.data.meta.last_page,
        };
```

```javascript
commit('doneGetArticleMeta', articleMeta);
```

```javascript
// mutation

doneGetArticleMeta(state, articleMeta) {
  state.page = { ...articleMeta };
},
```

Reply:
https://github.com/gizumo-education/gizumo_wiki_lesson/pull/147/commits/685bf4c124bee5604a858d9a36a8efa7b94a4015